cms/tests/test_cache.py

```diff
@@ -363,8 +365,10 @@ def test_expiration_cache_plugins(self):
             self.assertNotEqual(response['Cache-Control'], cache_control1)
             # However, the Expires timestamp will be the same
             self.assertEqual(response['Expires'], expires1)
-            # As will the Last-Modified timestamp.
-            self.assertEqual(response['Last-Modified'], last_modified1)
+            # No longer set / needed by Django 1.11
```

Comment:
if this is no longer needed on 1.11, can we remove it all-together? Seems like this check was not necessary.